packages/salesforcedx-vscode-apex-debugger/README.md

```diff
@@ -1,3 +1,146 @@
+            {
+                "name": "Launch Apex Debugger",
+                "type": "apex",
+                "request": "launch",
+                "sfdxProject": "${workspaceRoot}"
+            }
+        ]
+        ```
+    1. Save your `launch.json` file.
+
+## Debug Your Code
+After you’ve set up the Apex Debugger, set breakpoints and start a debugging session. Then, debug your code.
+
+To set a breakpoint, open a `.cls` or `.trigger` file and click the column to the left of the line numbers. Active breakpoints are red. Inactive breakpoints are grey. You can see a list of your breakpoints in the Breakpoints panel of the Debug view.
+
+To start a debugging session, from the configuration dropdown menu at the top of the Debug view, select **Launch Apex Debugger**. Then, click the green play icon (hover text: Start Debugging).
+
+While a debugging session is in progress, any synchronous activity that runs a line of code with a breakpoint causes execution to halt at the breakpoint. While execution is paused, you can inspect the call stack and see the current values of your variables. You can also step through your code, using the Debug actions pane that appears at the top of the editor while a debugging session is in progress, and watch those values change. You can debug up to two threads at a time. For more information, see [Debugging](https://code.visualstudio.com/docs/editor/debugging) in the Visual Studio Code Docs.
+
+## Whitelist Users and Request Types
+To filter which requests are debugged, edit your `launch.json` file to set up whitelisting. If you don’t use whitelisting, all events in your org trigger debugging during a debugging session. Whitelist users or request types to focus only on the events that are relevant to the problem you’re debugging.
+
+Add filters to the `"Launch Apex Debugger"` configuration:  
+```
+    "configurations": [
+        {
+            "name": "Launch Apex Debugger",
+            "type": "apex",
+            "request": "launch",
+            "sfdxProject": "${workspaceRoot}"
+            "userIdFilter": "",
+            "requestTypeFilter": "",
+            "entryPointFilter": ""
+        }
+    ]
+```
+
+To auto-complete potential request type values for `"requestTypeFilter"`, press Ctrl+Space.
+
+To filter by entry point, enter a regular expression as the value for `"entryPointFilter"`. For example, to whitelist requests made by the Visualforce page `MyPage`, enter `".*/apex/MyPage.apexp"`.
+
+## Considerations
+Keep these limitations and known issues in mind when working with the Apex Debugger.
+
+* If you edit Apex classes while a debugging session is in progress, your breakpoints might not match your debugging output after you save your changes.  
+
+* Your debugging session is orphaned when you close VS Code before stopping your session. If you have an orphaned session, you can’t start a new session. To manage your Dev Hub’s Apex Debugger sessions, go to Apex Debugger in Setup.  
```

Comment:
There's a command palette item labeled "SFDX: Stop Apex Debugger Session" that will query for active debugger session in their scratch org and terminate it, if they don't want to go to the Setup page. They will need salesforcedx-vscode-core extension installed.